Rocket.Chat/NewRoomViewController.swift

```diff
@@ -169,11 +169,15 @@ class NewRoomViewController: BaseViewController {
                     completion(false, result.error)
                     return
             }
+            guard let name = result.name else {
```

Comment:
Since the handler block is the same, you can merge this check with the previous guard, breaking each verification per line. What do you think?